packages/graphback-runtime/src/data/KnexDBDataProvider.ts

```diff
@@ -16,24 +18,31 @@ import { NoDataError } from './NoDataError';
 export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
 
   protected db: Knex;
+  protected baseType: GraphQLObjectType;
+  protected tableName: string;
 
-  constructor(db: Knex) {
+  constructor(baseType: GraphQLObjectType, db: Knex) {
     this.db = db;
+    this.baseType = baseType;
```

Comment:
I wonder what the implications of only having a single model passed to the provider. Will we need information about relationship models etc which may not be possible to get from single object?

Reply:
Object will contain relationships information as it will be embeeded into the type.getFields()[0].getType(). 

In fact we will need to check if the type is not custom scalar or union so we should always based on this. 
There are couple questions about relationships. 

If the relationship type has no model annotation it should probably be ignored.